commons/connector/upload/altoida/altoida_ar_test.avsc

```diff
@@ -0,0 +1,38 @@
+{
+    "namespace": "org.radarcns.connector.upload.altoida",
+    "name": "AltoidaARTest",
+    "type": "record",
+    "doc": "Schema of events for the Altoida AR tests.",
+    "fields": [
+      { "name": "time", "type": "double", "doc": "Timestamp in UTC (s) when the assessment is started by the subject." },
+      { "name": "timeReceived", "type": "double", "doc": "Timestamp in unix time received from Altoida API."},
+      { "name": "name", "type": "string", "doc": "Questionnaire name." },
+      { "name": "buttons", "type": {
+        "type": "array",
+        "items": {
+          "name": "ARButton",
+          "type": "record",
+          "doc": "AR Test button / screen element.",
+           "fields": [
+             { "name": "name", "type": "string", "doc": "Element name."},
+             { "name": "xCenterOffset", "type": "double", "doc": "Element location x center offset in cm." },
+             { "name": "yCenterOffset", "type": "double", "doc": "Element location y center offset in cm." },
+             { "name": "width", "type": "double", "doc": "Element width in cm." },
+             { "name": "height", "type": "double", "doc": "Element height in cm." }
+           ]
+         }}, "doc": "Buttons / screen elements list."},
+      { "name": "events", "type": {
+        "type": "array",
+        "items": {
+          "name": "AssessmentEvent",
```

Comment:
@blootsvoets There are a few schemas that have the `events` (`AssessmentEvent`) array data. Should they just share the same data type or is it fine that they're separate?

Reply:
If they have the same elements (or very similar), it's fine to make it a shared object. Preferably in a separate file as well then.